lib/MusicBrainz/Server/Form/User/Register.pm

```diff
@@ -10,7 +11,8 @@ has '+name' => ( default => 'register' );
 has_field 'username' => (
     type      => 'Text',
     required  => 1,
-    maxlength => 64
+    maxlength => 64,
+    validate_method => \&validate_username
```

Comment:
It's def. okay to add the trailing commas in new lines or lines you're changing anyway, even if other parts of the file don't use it. But, it's not like I'll reject a PR just 'cause it didn't have trailing commas*—it's just better to add them.

*If we checked code style in the tests using e.g. Perl::Critic, it'd be much easier to say "here, fix this list of issues if you want the build to pass," than having to make tedious comments on GitHub which usually aren't worth my time (and not because I don't care).